### PR TITLE
2791 icon name mapping

### PR DESCRIPTION
**Pull request by harshil1793**
## Description
This PR enhances the ESLint migration script to automatically map Font Awesome icon names to their corresponding USWDS icon names for the `va-icon` component. By leveraging the provided icon name mapping chart, the script will now update the icon prop with the appropriate USWDS icon names during the migration process.

## Testing done
Verified that the migration script correctly updates the icon prop with USWDS icon names.

## Screenshots
Warning:
Screenshot 2024-06-21 at 4 32 44 PM

Fix:
Screenshot 2024-06-21 at 4 33 19 PM

## Acceptance criteria
- [ ] Given a provided Font Awesome class name, the mapped USWDS icon name populates for the icon prop in the `va-icon` migration script

## Definition of done
- [ ] Changes have been tested in vets-website
- [ ] Changes have been tested in IE11, if applicable
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by micahchiang — COMMENTED** [review]
This looks great, thanks Harshil! Were you able to test this locally in vets-website using verdaccio?

**Comment by harshil1793** [comment]
@micahchiang Yes, I tested that in vets-website. I have verified it again and attached the screenshots.

**Review by it-harrison — APPROVED** [review]

**Comment by jamigibbs** [comment]
https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2791

File: packages/eslint-plugin/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/eslint-plugin",
-  "version": "1.18.1",
+  "version": "1.18.2",
```
Comment (jamigibbs) [inline packages/eslint-plugin/package.json:3]: @harshil1793 Were you able to create a new release for this package yet and update vets-website?

**Pull request merged**